### ISSUE TITLE
Suggestion for the button to translate the selected text

### ISSUE THREAD

**Issue by VladimirCo** [FilipePS/Traduzir-paginas-web#87]
When the button to translate the selected text appears on the page it would be very convenient if the function of translating the selected text had the ability not only to translate a foreign language into the native language - BUT also translate the native language into a foreign one simply by highlighting the text and click to "Translate to English (for example)"

**Comment by VladimirCo** [comment]
Better yet, use the Keyboard shortcut to translate the selected text, for example double-clicking CTRL, replacing the translated text 
This is done very conveniently in the program, in the translator section: https://everylang.net/help?%2Fhelp= 
Сopy the translated text and replace the selected text, for clarity, look at the screenshot:  
![Screenshot](https://user-images.githubusercontent.com/17863616/107362593-a0a0a300-6ae1-11eb-8f15-8646a34de5eb.png)

**Comment by Piulinux** [comment]
To summarize:
Add the possibility to change the language to be translated to right when you have selected something.
Add the possibility to replace the selected text with the translation (and add a keyboard shortcut for that).

The TTS feature would be awesome, too. (both TTS the original selected text and TTS the translation)

**Comment by FilipePS** [comment]
> Add the possibility to change the language to be translated to right when you have selected something.
> The TTS feature would be awesome, too. (both read the original selected text and read the translation)

This was added in version 8.6

> Add the possibility to replace the selected text with its translation (and add a keyboard shortcut for that).

Are you saying to change the text of the page by what you type, as if it were a translation of the page, but manual?
Or the translated text should appear only in the selected text popup.

**Comment by Piulinux** [comment]
> > Add the possibility to replace the selected text with its translation (and add a keyboard shortcut for that).
>
> Are you saying to change the text of the page by what you type, as if it were a translation of the page, but manual?
> Or the translated text should appear only in the selected text popup.

Yes, the text you have selected or hovered on should be replaced with the translation, if I get @VladimirCo correctly:

> Better yet, use the Keyboard shortcut to translate the selected text, for example double-clicking CTRL, replacing the translated text

**Comment by VladimirCo** [comment]
> Yes, the text you have selected or hovered on should be replaced with its translation, if I get @VladimirCo correctly:

Yes, you understood me correctly. Should be replaced with translated text instead of typed text. 
   @FilipePS , for example your native language is Spanish on you quickly think and type in it (because it is your native language) and you want the typed text to be quickly translated into English (for example).  In this case, you write in Spanish -> select this typed text -> press double-clicking  CTRL (the hotkey translation the selected text) -> a window appears with the translated selected text (after pressing will pоp up а window with the translation text) -> click on the button "insert translated text instead of typed text". 
  This function is already implemented in this program https://everylang.net/help?%2Fhelp=, above is the screen for its description.  Install this program for clarity to see how this function works and just copy this killer-feature because it is sooooo handy.

P.S.: The update of version 8.6 is good - I already made a donation as a thank you :)
